templates/case-study/grundium-ubuntu-pro-for-devices.html

```diff
@@ -0,0 +1,202 @@
+{% extends "case-study/base_case-study.html" %}
+
+{% from "macros/_macros-hero.jinja" import hero %}
+{% from "macros/_macros-text-list.jinja" import text_list %}
+{% from "macros/_macros-highlights.jinja" import highlights %}
+{% from "macros/_macros-text.jinja" import text %}
+{% from "macros/_macros-quote.jinja" import quote_wrapper %}
+{% from "macros/_macros-image.jinja" import image_wrapper %}
+{% from "macros/_macros-cta.jinja" import cta %}
+
+{% block title %}Grundium secures critical systems with Ubuntu Pro for Devices{% endblock %}
+
+{% block meta_description %}
+  As a medical device manufacturer, Grundium must meet stringent security requirements. Ubuntu Pro for Devices allows them to secure their digital microscopes.
+{% endblock %}
+
+{% block meta_company_name %}Grundium{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1wM1vo24SFowPaAnxWV9wfU-w6PVDq7RczM_5ZOb8OZM/edit
+{% endblock %}
+
+{% block content %}
+  {%- call(slot) hero(title="Canonical deploys infrastructure solutions and managed IT services for critical space mission operations",
+                      layout="25/75",
+                      is_split_on_medium="true") -%}
+    {% if slot == "signpost_image" %}
+      {{ image(url="https://assets.ubuntu.com/v1/547c1ef8-Grundium-logo-black 1.png",
+            alt="",
+            width="264",
+            height="53",
+            hi_def=True,
+            loading="auto") | safe
+      }}
+    {% endif %}
+  {%- endcall -%}
+
+  {%- call(slot) text_list(title="About Grundium") -%}
+    {%- if slot == 'list_item_content_1' -%}
+      Grundium is a medical device manufacturer that specialises in digital microscopy. Their range of microscope scanners are known as Ocus® Scanners.
```

Comment:
```suggestion
      Grundium is a medical device manufacturer that specialises in digital microscopy. Their range of microscope scanners are known as Ocus&reg; Scanners.
```
Please use &reg; in place of the symbol